code/modules/mob/living/simple_animal/hostile/terror_spiders/chem.dm

```diff
@@ -0,0 +1,101 @@
+#define SOLID 1
+#define LIQUID 2
+#define GAS 3
+#define FOOD_METABOLISM 0.4
+#define REM REAGENTS_EFFECT_MULTIPLIER
+
+
+// Terror Spider, Black, Deadly Venom
+
+/datum/reagent/terror_black_toxin
+	name = "Black Widow venom"
+	id = "terror_black_toxin"
+	description = "An incredibly toxic venom injected by the Black Widow spider."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 0.1
+
+/datum/reagent/terror_black_toxin/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
```

Comment:
Single line control statements are against the style guide, largely because they're less readable than multi-line ones.